ide/vagrant/Vagrantfile

```diff
@@ -45,6 +45,11 @@ Vagrant.configure("2") do |config|
             # meminfo shows KB and we need to convert to MB
             mem = `grep 'MemTotal' /proc/meminfo | sed -e 's/MemTotal://' -e 's/ kB//'`.to_i / 1024 / 2
 
+        elsif Vagrant::Util::Platform.windows?
+
+            cpus = `wmic cpu get NumberOfCores`.split("\n")[2].to_i
+            mem = `wmic OS get TotalVisibleMemorySize`.split("\n")[2].to_i / 1024 /4 / 2
+
```

Comment:
Can you be more specific in your commit message?

Reply:
Sure, will do!